src/main/java/ru/endlesscode/rpginventory/event/listener/CraftListener.java

```diff
@@ -74,13 +78,26 @@ public void onPacketSending(PacketEvent event) {
         }
     }
 
+    @EventHandler(priority = EventPriority.LOWEST)

```

Comment:
Use ``EventPriority.LOW`` instead of ``EventPriority.LOWEST``

Reply:
Ok, I'll fix it in next commit